testing/kuttl/e2e/switchover/01--cluster.yaml

```diff
@@ -12,9 +12,3 @@ spec:
   instances:
     - replicas: 2
```

Comment:
Does the test run slower when backups are off and replicas > 1?

Reply:
Good question, I see four tests that have replicas and need them, so I'm testing them out:
* replica-read
  * w/ backups:  ~28s
  * w/o backups:~25s
* scaledown
  * w/ backups:  ~2m
  * w/o backups:~3m
* switchover
  * w/ backups:  ~37s
  * w/o backups:~34s
 
 So adding backups back to Scaledown files for now and testing.